plugins/modules/helm_cli_info.py

```diff
@@ -0,0 +1,290 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: helm_cli_info
+short_description: Get informations from Helm package deployed inside the cluster
+description:
+  -  Get informations (values, states, ...) from Helm package deployed inside the cluster
+version_added: "2.10"
+author:
+  - Lucas Boisserie (@LucasBoisserie)
+requirements:
+  - "helm (https://github.com/helm/helm/releases)"
+  - "yaml (https://pypi.org/project/PyYAML/)"
+options:
+  binary_path:
+    description:
+      - The path of a helm binary to use.
+    required: false
+    type: path
+  release_name:
+    description:
+      - Release name to manage
+    required: true
+    type: str
+    aliases: [ name ]
+  release_namespace:
+    description:
+      - Kubernetes namespace where the chart should be installed
+      - Can't be changed with helm 2
+    default: "default"
```

Comment:
Just for consistency, the other modules don't set a default for namespace and instead set it as required. I think it makes sense to be explicit since most CLI utilities don't default to `default`, but rather a namespace determined by the current context, and since we're not matching that behavior defaulting to a specific namespace could trip up some users.

Reply:
Ok, i will change that